examples/ios/Mixed/SwiftGreeter.swift

```diff
@@ -13,6 +13,7 @@
 // under the License.
 
 import Foundation
+import submodule
```

Comment:
Can you try again with this change? It worked for me.

```suggestion
@_implementationOnly import submodule
```

Reply:
Yes!! That works.   Where can I find more information about these cool @_ swift tricks?

What do you want me to do with this pr?

Reply:
Probably the Swift forums: https://forums.swift.org/t/update-on-implementation-only-imports/26996.

Could you make this a separate example (e.g. MixedWithPrivateSubmodule)? Then I'll merge it.

Reply:
done